code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -1,22 +1,24 @@
 /mob/living/simple_animal/hostile/netherworld
 	name = "creature"
 	desc = "A sanity-destroying otherthing from the netherworld."
-	icon_state = "otherthing"
-	icon_living = "otherthing"
-	icon_dead = "otherthing-dead"
+	icon_state = "otherthing-pink"
+	icon_living = "otherthing-pink"
+	icon_dead = "otherthing-pink-dead"
 	health = 80
 	maxHealth = 80
 	obj_damage = 100
 	melee_damage_lower = 25
 	melee_damage_upper = 50
-	attacktext = "slashes"
+	attacktext = "chomps"
```

Comment:
No reason for this

Reply:
I said this one is fine to fit the new sprite, in his defense

Reply:
Ah I got it, can resolve then.